app/helpers/redirect_destination_helper.rb

```diff
@@ -1,7 +1,7 @@
 module RedirectDestinationHelper
   def redirect_destination_select_options
     guide_select_options = Guide
-      .with_published_editions
+      .live
```

Comment:
Sounds like an accident. Probably mine. We agreed on this project to use leading dots so that's what they should be.

Neither the alphagov or the bbatsov style guide have a preference although the latter talks about it: https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains

My addition to the argument is that trailing dots let you copy and paste stuff into irb/console. With a leading dot it doesn't know if there is another chained method coming and so usually errors.